.github/workflows/Tests.yaml

```diff
@@ -66,6 +66,22 @@ jobs:
           cd $GITHUB_WORKSPACE
           ./tools/CheckFiles.sh
 
+  # Run format checker for python to make sure the code is pep8 conforming
```

Comment:
better remove the `pep8 conforming` I think, since `yapf` only does formatting but pep8 guidelines are also about naming conventions and such. `yapf` can't do all that (and doesn't try 😄)

Reply:
ping on this